src/components/TodoFilter/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoFilter';
```

Comment:
It's recommended to use a clear way to import and export to maintain consistency. Instead of directly exporting from an import, consider importing the module and then exporting it. 🌟